### PR TITLE
Init commit

### PR DESCRIPTION
**Pull request by Scriptkiddi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# Syncrify-Server
```
Comment (dasJ) [inline README.md:1]: Commit message…

File: README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+## Requirements
+
+An apt based system
+
+## Role Variables
+
+| Name                                              | Required/Default                            | Description                                  |
+|---------------------------------------------------|:-------------------------------------------:|----------------------------------------------|
+| `syncrify_server_admin_email`                     | :heavy_check_mark:                          |                                              |
```
Comment (dasJ) [inline README.md]: So we don't do descriptions anymore?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+| `syncrify_server_preferred_host_name_for_clients` | :heavy_check_mark:                          |                                              |
+| `syncrify_server_smtp_security`                   | `None`                                      | TLS/SSL/None                                 |
```
Comment (dasJ) [inline README.md]: This is not a description

File: README.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+| `syncrify_server_smtp_port`                       | `587`                                       | Smtp port                                    |
```
Comment (dasJ) [inline README.md]: It's called SMTP. Also, what is it for? This not a description either

File: README.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+| `syncrify_server_smtp_server`                     | `null`                                      | Url of the smtp server                       |
```
Comment (dasJ) [inline README.md]: Yes but what for

File: README.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+| `syncrify_server_smtp_user`                       | `null`                                      | Smtp username                                |
+| `syncrify_server_smtp_password`                   | `null`                                      | Password for the smtp user                   |
+| `syncrify_server_use_web_smtp`                    | `false`                                     | Enable web smtp                              |
```
Comment (dasJ) [inline README.md]: So… SMTP over HTTP? Or what?

File: README.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+| `syncrify_server_email_server_web_service_host`   | `null`                                      | Web email service host url                   |
```
Comment (dasJ) [inline README.md]: Excuse me?

File: README.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+| `syncrify_server_email_server_web_service_port`   | `null`                                      | Port for web email                           |
+| `syncrify_server_use_rdbms_for_delete_retension`  | `true`                                      | Enable or disable rdbms retension            |
```
Comment (dasJ) [inline README.md]: Pretty sure you mean retention. Retension is something different

File: README.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+| `syncrify_server_jvm_path`                        | `jre/bin/java`                              | Path where the java binary is located        |
+| `syncrify_server_http_ip`                         | `127.0.0.1`                                 | Ip address to bind to                        |
+| `syncrify_server_http_port`                       | `0`                                         | Http port to listen on (0 disables the port) |
+| `syncrify_server_http_port2`                      | `5800`                                      | Http port to listen on                       |
+| `syncrify_server_http_port_ssl`                   | `-1`                                        | SSL Port -1 disables the port                |
```
Comment (dasJ) [inline README.md]:
```suggestion
| `syncrify_server_http_port_ssl`                   | `-1`                                        | SSL Port. A value of `-1` disables the port                |
```

File: README.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+| `syncrify_server_vm_params`                       | `-Xmx512m -DLoggingConfigFile=logconfig.xm` | Java VM parameters                           |
+| `syncrify_server_synametrics_url`                 | `http://synametrics.com/SynametricsWebApp/` | Synametrics URl                              |
+| `syncrify_server_last_selected_tab`               | `1`                                         | Which tab was last selected                  |
```
Comment (dasJ) [inline README.md]:
```suggestion
| `syncrify_server_last_selected_tab`               | `1`                                         | Tab which was last selected                  |
```

Comment (dasJ) [reply README.md]: Also. What?

File: README.md [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+| `syncrify_server_image_path`                      | `images/`                                   | Images path                                  |
+| `syncrify_server_block_rdbms_to_handlers`         | `true`                                      | Block or allow handlers to rdbms             |
+| `syncrify_server_default_operation`               | `door`                                      | Default operation                            |
+| `syncrify_server_ransomware_file_name`            | `6LUtvYHu.jpg`                              | Filename for ransomeware check               |
+| `syncrify_server_failure_over_http_port`          | `54222`                                     | Failover http port                           |
+| `syncrify_server_disable_csrf_prevention`         | `true`                                      | Disable or enable csrf prevention            |
+| `syncrify_server_cd`                              | `1565091689911`                             |                                              |
```
Comment (dasJ) [inline README.md]: ????

File: README.md [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+| `syncrify_server_nt_service_command`              | `systemctl start syncrify.service`          | The nt service command                       |
```
Comment (dasJ) [inline README.md]: Okay.

File: README.md [hunk 13]
```diff
@@ -0,0 +1,83 @@
...
+| `syncrify_server_mimic_html_files`                | `false`                                     | Mimic html files                             |
```
Comment (dasJ) [inline README.md]: Pretty sure it's called HTML

File: README.md [hunk 14]
```diff
@@ -0,0 +1,83 @@
...
+  - domains:
+      - backup.example.de.
+    default_server: true
+    client_max_body_size: 50m
+    crypto: true
+    https: true
+    locations:
+      - condition: ~* ^/
+        content: |
+          root /opt/Syncrify/htdocs/webapps/ROOT;
```
Comment (dasJ) [inline README.md]: Maybe mention this path at the top of the readme

File: README.md [hunk 15]
```diff
@@ -0,0 +1,83 @@
...
+          try_files $uri $uri/ @backend;
+      - condition: "@backend"
+        content: |
+          proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+          proxy_set_header X-Forwarded-Proto https;
+          proxy_set_header Host $http_host;
+          proxy_pass http://backend;
```
Comment (dasJ) [inline README.md]: Why use an upstream in the first place when only one host is specified?

File: meta/main.yml [hunk 16]
```diff
@@ -0,0 +1,19 @@
+---
+galaxy_info:
+  author: Insert author(s) here
```
Comment (dasJ) [inline meta/main.yml]: Did you even try

File: tasks/main.yml [hunk 17]
```diff
@@ -0,0 +1,68 @@
+---
+- name: Install java
+  apt:
+    name: openjdk-11-jre
```
Comment (dasJ) [inline tasks/main.yml:4]: Why not default to the JRE you install here?

File: tasks/main.yml [hunk 18]
```diff
@@ -0,0 +1,68 @@
...
+
+- name: Check if syncrify is installed
+  stat:
+    path: /opt/Syncrify
+  register: syncrify_folder
+
+- name: Download syncrify server
+  get_url:
+    url: http://www2.synametrics.com/files/Syncrify/SyncrifyLinux64.tar
+    dest: /tmp/syncrify-server.tar
```
Comment (dasJ) [inline tasks/main.yml]: Forgot about global_cache_dir?

File: tasks/main.yml [hunk 19]
```diff
@@ -0,0 +1,68 @@
...
+  when: not syncrify_folder.stat.exists
+
+- debug:
+    msg: "The file or directory exists"
+  when: syncrify_folder.stat.exists
```
Comment (dasJ) [inline tasks/main.yml]: Why?

File: tasks/main.yml [hunk 20]
```diff
@@ -0,0 +1,68 @@
...
+  when: not syncrify_folder.stat.exists
+
+- name: Unarchive syncrify files
+  unarchive:
+    src: /tmp/syncrify-server.tar
+    dest: /tmp/syncrify
+    remote_src: yes
+  when: not syncrify_folder.stat.exists
+
+- name: Set executable for Install.sh
```
Comment (dasJ) [inline tasks/main.yml]:
```suggestion
- name: Set executable bit for Install.sh
```

File: tasks/main.yml [hunk 21]
```diff
@@ -0,0 +1,68 @@
...
+- name: Run install script
+  command: sh Install.sh
+  args:
+    chdir: /tmp/syncrify/
+  when: not syncrify_folder.stat.exists
+
+- name: Place AppConfig.xml
+  template:
+    src: AppConfig.xml.j2
+    dest: /opt/Syncrify/config/AppConfig.xml
```
Comment (dasJ) [inline tasks/main.yml:46]: So no permission stuff?

File: tasks/main.yml [hunk 22]
```diff
@@ -0,0 +1,68 @@
...
+
+- name: Systemd service
```
Comment (dasJ) [inline tasks/main.yml]: That's not a really helpful name

File: templates/syncrify.service.j2 [hunk 23]
```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Service for Syncrify Start on boot
+After=network.target
+
+[Service]
+Type=forking
+ExecStart=/etc/init.d/syncrify start
```
Comment (dasJ) [inline templates/syncrify.service.j2]: Is there *any* reason to run this as root?

**Review by dasJ — CHANGES_REQUESTED** [review]

**Comment by Scriptkiddi** [comment]
@dasJ please review

**Comment by Scriptkiddi** [comment]
@dasJ

**Comment by dasJ** [comment]
Automatically bumping this issue because it was inactive for 7 days.

**Comment by Scriptkiddi** [comment]
@dasJ Automatically bumping this issue because it was inactive for 7 days. ja meinsch

**Pull request closed without merging**